tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/confirm_install.py

```diff
@@ -21,3 +21,4 @@ def tap_confirm(self):
                 *self._confirm_install_button_locator))
             Wait(self.marionette).until(expected.element_displayed(confirm))
             confirm.tap()
+            self.apps.switch_to_displayed_app()
```

Comment:
I think there should be a separate PR for moving the file in. I did remove the sleep call.